pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -673,6 +673,10 @@ self: super: {
     dependencies = with self; [ plenary-nvim ];
   });
 
+  nvim-teal-maker = super.nvim-teal-maker.overrideAttrs (old: {
+    buildInputs = [ luaPackages.cyan luaPackages.tl ];
```

Comment:
I don't think this works as expected, I get this when trying to run `TealBuild`

> E5108: Error executing lua ...vimPackages/start/nvim-teal-maker/lua/tealmaker/init.lua:149: Could not find 'cyan' on path.  This is necessary to build teal files

Reply:
indeed, I had not tested it yet sry xD It works now. My recommendation for Integration with nvim is:
- tlconfig.lua should go in ~/.config/nvim (was not clear for me) 
- add `require("tealmaker").build_all(verbose_output) ` in your init.lua 

To make it work, https://github.com/teal-language/teal-types is needed too but integrating it is not yet straightforward. Might be better in a home-manager module (that's how I do it).

Reply:
I think it's fine to require extra set up since patching seems non-trivial